data/_models/shared/Processes.csv

```diff
@@ -58,7 +60,7 @@ Clinical observations,Observations,,,,,,,,,,,http://purl.obolibrary.org/obo/NCIT
 Clinical observations,,observation performed by,ref,,,,Contacts,,,,,http://purl.obolibrary.org/obo/NCIT_C48206,"The individual that performed the clinical observation (e.g., made the diagnosis, performed the evaluation, etc.)","Patient registry, FAIR Genomes"
 Clinical observations,,phenotypes,refback,,,,Phenotype observations,,part of clinical observation,,,http://purl.obolibrary.org/obo/NCIT_C16977,"The assemblage of traits or outward appearance of an individual. It is the product of interactions between genes and between genes and the environment. In medical context,  these are often the symptoms caused by a disease.","Patient registry, FAIR Genomes"
 Clinical observations,,diseases,refback,,,,Disease history,,part of clinical observation,,,http://purl.obolibrary.org/obo/NCIT_C15607,"Any abnormal condition of the body or mind that causes discomfort, dysfunction, or distress to the person affected or those in contact with the person. The term is often used broadly to include injuries, disabilities, syndromes, symptoms, deviant behaviors, and atypical variations of structure and function.","Patient registry, FAIR Genomes"
-Clinical observations,,measurements,refback,,,,Measurements,,part of clinical observation,,,http://purl.obolibrary.org/obo/NCIT_C25209,Annotation used to indicate the size or magnitude of something that was determined by comparison to a standard,"Patient registry, FAIR Genomes"
+Clinical observations,,measurements,refback,,,,Clinical measurements,,part of clinical observation,,,http://purl.obolibrary.org/obo/NCIT_C25209,Annotation used to indicate the size or magnitude of something that was determined by comparison to a standard,"Patient registry, FAIR Genomes"
```

Comment:
I think we can use the definition for clinical measurement here (as listed in line 91).